<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FDialateDialog</class>
 <widget class="QDialog" name="FDialateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialate</string>
  </property>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cross</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblSizeX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Size X</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSizeY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnSizeX">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnSizeY">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOk">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
